egs/gale_mandarin/s5/local/split_wer_per_corpus.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This new script is not called in run.sh. We may think of deprecate the original split_wer.sh and rename this one as split_wer.sh. That one was errornous anyway.